pkg/awx/connection.go

```diff
@@ -345,7 +348,14 @@ func (c *Connection) rawPost(path string, query url.Values, input []byte) (outpu
 	if glog.V(2) {
 		glog.Infof("Response body:\n%s", c.indent(output))
```

Comment:
If we are using `httputil.DumpRequest` for the request we should also use `httputil.DumpResponse` for the response.